src/components/Rocketcomponent.js

```diff
@@ -7,14 +7,21 @@ const Rocket = (props) => {
   const {
     imgsrc, imgname, rocketname, rocketdescription, id, reserved,
   } = props;
+  let reserveBadge = '';
+  if (reserved) {
+    reserveBadge = <span className={style.badge}>Reserved</span>;
+  }
```

Comment:
I only can click to reserve. The cancel reservation is not working. Kindly fix it.

![re](https://user-images.githubusercontent.com/72808808/132514368-fa9e6311-4279-4d7f-9e5f-a71750ebf7ae.png)

Thank you